sh/tests/to-git-or-not-to-git_test.sh

```diff
@@ -8,6 +8,10 @@ FILENAME="student/to-git-or-not-to-git.sh"
 
 if [ -f ${FILENAME} ]; then
     if [ -s ${FILENAME} ]; then
+        submitted=$(bash student/to-git-or-not-to-git.sh)
+        expected=$(bash solutions/to-git-or-not-to-git.sh)
+        echo $submitted
+        echo $expected
         diff <(echo "$submitted") <(echo "$expected")
     else
         echo "The file exist but empty"
```

Comment:
Should there be an `exit 1` after this?
https://github.com/01-edu/public/blob/87dca1cf80f4339053494f39902b8ba179f1d067/sh/tests/to-git-or-not-to-git_test.sh#L21

This appears to exit success when the file does not exist. Or does the platform work differently to that?

Sorry, I can't actually comment on line 21.

Reply:
i guess it doesn't do  nothing there, so i removed it.
The testing for this kind of exercise is tricky on the platform... We have to merge the test in master in order to test it on the platform because we use variables like (DOMAIN) and (USERNAME) so it can only be properly tested after its merged.
But i tested using a fix domain and username and it was working good so i assume it will be ok

Reply:
I'm pretty sure that an `exit 1` must follow both of these so that the platform can detect failure:
https://github.com/01-edu/public/blob/5e6e56dfe1e500fc29589cb25d7b2f845b9b2ab3/sh/tests/to-git-or-not-to-git_test.sh#L18
https://github.com/01-edu/public/blob/5e6e56dfe1e500fc29589cb25d7b2f845b9b2ab3/sh/tests/to-git-or-not-to-git_test.sh#L21

The other shell tests appear to work in that way.

Reply:
fixed it, thanks for the clarification!